Assets/Gothic-UnZENity-Core/Scripts/GameConfiguration.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using GUZ.Core.Context;
+using GUZ.Core.World;
+using UnityEngine;
+
+namespace GUZ.Core
+{
+	[Serializable]
+	public class MeshCullingGroup
+	{
+		[Range(1f, 100f)] public float maximumObjectSize;
+		[Range(1f, 1000f)] public float cullingDistance;
+	}
+
+	[CreateAssetMenu(fileName = "Data", menuName = "ScriptableObjects/GameConfiguration", order = 1)]
+	public class GameConfiguration : ScriptableObject
+	{
+		[Header("### Controls ###")] public bool enableDeviceSimulator = false;
+        public GUZContext.Controls gameControls = GUZContext.Controls.VR_XRIT;
+
+		[Header("### Developer ###")] [SerializeField]
+		public bool enableMainMenu = true;
+
+		[SerializeField] public bool spawnOldCampNpcs = false;
+		[SerializeField] public bool enableNpcRoutines = false;
+
+		[SerializeField] public string spawnAtWaypoint = string.Empty;
+
+		[SerializeField] public bool enableWorldObjects = true;
+		[SerializeField] public bool enableWorldMesh = true;
+		[SerializeField] public bool enableBarrierVisual = true;
+		
+		[InspectorName("Experimental: Enable Decal Visuals")]
+		[SerializeField] public bool enableDecalVisuals = true;
```

Comment:
Should be: _false_ by default / for Production builds.
Hint: If you don't want to reflect "production" in the class itself, then let's stick with C#'s default values in general and we always _clone_ the Production.asset one whenever we need a specific instance.
(I won't comment more elements on this for now. I think the idea with C# default values would help. Let's discuss.)